src/components/shared/SecondaryNav/index.tsx

```diff
@@ -1,38 +1,38 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React, { ReactNode, ReactNodeArray } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import classnames from 'classnames';
 
 import './index.scss';
 
 type SecondaryNavProps = {
-  secondaryNavList?: any[];
-  activeNavItem?: string | undefined;
+  children: ReactNode | ReactNodeArray;
```

Comment:
(nit) I _think_ this can just be:
```suggestion
  children: ReactNode;
```